addons/web/static/src/js/model.js

```diff
@@ -57,33 +263,123 @@ odoo.define('web.model', function () {
         /**
          * Call the base model method with given name with the arguments
          * determined by the dispatch extra arguments.
-         *
-         * @param {string} action
-         * @param {...any} args
-         * @returns {Promise<any>}
+         * @returns {Promise}
          */
-        async dispatch(action, ...args) {
-            const result = await this[action](...args);
-            // TODO try to put it in promise.resolve()
-            this._dispatch(...arguments);
+        async dispatch(method, ...args) {
+            // Phase 1: call dispatch methods
```

Comment:
The dispatch results are no more returned --> problem in dashboard related to updateFilters in control_panel_model.js

Reply:
done